src/PKSim.Infrastructure/ORM/Repositories/OntogenyRepository.cs

```diff
@@ -176,8 +181,10 @@ public DistributedTableFormula DataRepositoryToDistributedTableFormula(DataRepos
 
       public DistributedTableFormula OntogenyToDistributedTableFormula(Ontogeny ontogeny, string containerName)
       {
+         //We use a clone in this case as we do want to make sure we do not have two difference two references to the same table 
+         //this would lead otherwise as the same table being referenced by a parameter and the ontogeny and an issue will arise when loading
```

Comment:
_we do not have two difference two references to the same table_ you mean "we do not have two references to the same table"?
_this would lead otherwise as the same table being referenced by_ something is missing here.